### DISCUSSION TITLE
one-to-many head

### DISCUSSION THREAD

**Discussion by MuhabHariri** [THU-MIG/yolov10#348] [Q&A]
Hi

Regarding this table :
![image](https://github.com/user-attachments/assets/fa298bec-87eb-4a4c-a228-f6ed011ccf64)

I noticed that you declared the number of FLOPs for YOLOv10n as 6.7G, but when I train it, I see that it is 8.7G. I searched in the issues section and found this answer from @jameslahm :

"Thanks for your interest! It seems that you also count the parameters and FLOPs of the one-to-many head. Since the one-to-many head is not needed during inference, this part of parameters and FLOPs can be ignored."

My question is: doesn't the same situation (the one-to-many head not being needed during inference) apply to YOLOv8n, which means the number of FLOPs should be lower for it too?

**Comment by matpalm** [comment]
it's only v10 that the two heads ( one-to-one and one-to-many ). v8 only uses the one.
